File: GO-BASIC/01.first-steps-in-coding/06.concatenate-data/concatenate-data.go
package main

import "fmt"

func main() {
	var fistName string
	var lastName string
	var age int
	var town string

	fmt.Scanln(&fistName)
	fmt.Scanln(&lastName)
	fmt.Scanln(&age)
	fmt.Scanln(&town)

	fmt.Printf("You are %s %s, a %d-years old person from %s.", fistName, lastName, age, town)
}

File: C++Basic/First-Steps-In-Coding-Lab/FirstStepsInCoding/concatenateData/concatenateData.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string name, lastName, town;
	cin >> name;
	cin >> lastName;
	
	int age;
	cin >> age;

	cin >> town;

	cout << "You are " << name << " " << lastName << ", a " << age << "-years old person from " << town << ".";
}